src/app/helptext/system/bootenv.ts

```diff
@@ -13,19 +16,15 @@ export const helptext_system_bootenv = {
 
   clone_name_placeholder: T("Name"),
   clone_name_tooltip: T(
-    "Enter a name for the clone of this boot environment.\
- Alphanumeric characters, dashes (*-*), underscores (*_*),\
- and periods (*.*) are allowed.)"
+    `Enter a name for the clone of this boot environment. ${allowedCharacters}`
```

Comment:
Unfortunately I don't think the translation extractor is smart enough to handle variables

Reply:
it won't work for the others either ;)

Reply:
It still won't work with the way you changed it, there is no way to engineer around this one unfortunately :-/

Reply:
I don't think I understand. All of our tooltips are variables in the component pointing to helptext. This one is two variables.